packages/app-desktop/gui/NotePropertiesDialog.tsx

```diff
@@ -300,12 +294,12 @@ class NotePropertiesDialog extends React.Component<Props, State> {
 		};
 
 		if (this.state.editedKey === key) {
-			if (key.indexOf('_time') >= 0) {
+			if (key.indexOf('_time') >= 0 || key === 'todo_completed') {
```

Comment:
(**Optional**): Since the `key.indexOf('_time') >= 0 || key === 'todo_completed'` check is used in multiple places, it may make sense to refactor it into a separate function.